frontend/src/components/ForumThread/ForumThreadPage.tsx

```diff
@@ -0,0 +1,81 @@
+import React, {useEffect, useState} from "react";
+import {ContentContainer} from "../App/AppStyle";
+import {ChatWindow, PostContent, PostDetails, SendButton, SenderArea, ThreadInfo} from "./ForumThreadStyle";
+import {useNavigate, useParams} from "react-router-dom";
+import {PageRoutes} from "../../utils/constants";
+import {Threads} from "./threadExample";
+import {ForumThread} from "../../api";
+import Loader from "../Loader/Loader";
+import {Col, Row} from "react-bootstrap";
+
+
+function getThread(id: number){
+    return Threads[id];
+}
+
+
+function fitAreaToContent(){
+    const maxHeight = 300; // px
+    let text = document.getElementById('text-field')!;
```

Comment:
Użyj useRef.
https://stackoverflow.com/questions/56920315/how-to-use-useref-to-change-the-style-of-a-element

Jak wpiszesz `ref={textFieldRef}` w propsach elementu i na to najedziesz IDE powinno ci podpowiedziec typ ktory mozesz przekopiowac.

Przykład:
```tsx
const textFieldRef: React.RefObject<HTMLTextAreaElement> = useRef(null);
...
<SenderArea ref={textFieldRef} onInput={() => textFieldRef.current!.style.height = "100px"} />
```

Reply:
Chyba gotowe, mam nadzieję że dobrze wykorzystałem tego hooka